### ISSUE TITLE
Nessus Parser

### ISSUE THREAD

**Issue by randomactsofsecurity** [sensepost/gowitness#69]
**Describe the solution you'd like**
A parser to support nessus files to be ran through gowitness, similar to nmap XML parser.  While you could use third party parsers to create a host file to be fun with the 'file' parser, could possibly add the support directly into gowitness.

**Describe alternatives you've considered**
Third party parsing libraries do already exist for nessus

**Additional context**
This crafty user has already written a golang nessus parser, maybe you could leverage it to pull the hosts & ports similar to nmap parser ;) 
https://gist.github.com/leonjza/bb977e98eac9565fae2aa177e498d9b8

**Comment by arch4ngel** [comment]
I agree that this would be useful. I may take a stab at integrating the idea. Not to peddle my own code, but this may help in the interim:

https://github.com/arch4ngel/parsuite

There's a module called `xml_dumper` that'll do what you're looking for. It works on both Nmap and Nessus files while retaining hostnames.

```
parsuite xml_dumper -ifs yourfile.nessus --format uri --fqdn --http-links
```

**Comment by leonjza** [comment]
This landed in e1923bb92e4e0935142e331225f8b903c396d509.
